.github/docker/linux/pr_build.sh

```diff
@@ -167,10 +173,10 @@ EOF
       *embed*)
         if [ -n "$do_valgrind" ]; then
           printf 'grinding agent unit tests\n'
-          make -r -s -j $(nproc) agent-valgrind "ARCH=${ARCH}" LDFLAGS='-Wl,--no-warn-search-mismatch -Wl,-z,muldefs'
```

Comment:
Why did you remove `LDFLAGS`?

Reply:
It would not compile with the flags.  This, along with the 32-bit/64-bit memory issue are still being investigated for issue #108 .